### DISCUSSION TITLE
pog

### DISCUSSION THREAD

**Discussion by HappyAreaBean** [diamond-rip/Eden#39] [General]
poggers
![圖片](https://user-images.githubusercontent.com/50882051/210363378-28a9ecbb-777e-44d2-9f60-eb1f8c8885cd.png)

**Comment by GoodestEnglish** [comment]
![image](https://user-images.githubusercontent.com/45838505/210363559-a6192c1e-fe6a-4e3d-b2b3-72948c68b8e2.png)

**Reply by HappyAreaBean** [reply]
holy shit this is a big peepo

**Comment by HierarchyMan** [comment]
Pog

**Reply by HappyAreaBean** [reply]
![圖片](https://user-images.githubusercontent.com/50882051/210363826-6c75a53c-d746-4ae4-9f4f-5f73748d2424.png)
